### PR TITLE
#11 Add Windows Support

### PR DESCRIPTION
**Pull request by XIAZY**

### CODE REVIEW

**Comment by XIAZY** [comment]
Azure pipeline ran out of space.
There's a 10 GB space limit for Microsoft-hosted pipelines. We may want to switch to a self-hosted CI server.

File: README.md [hunk 1]
```diff
@@ -74,6 +76,43 @@ To compile AlphaRTC, please refer to the following steps
...
+
+    ```shell
+    git clone https://github.com/OpenNetLab/AlphaRTC.git
+    ```
+
+3. Sync the dependencies
+    ```shell
+    cd AlphaRTC
+    gclient sync
+    mv -fvn src/* .
```
Comment (Pterosaur) [inline README.md:96]: This command doesn't work in Windows

Comment (XIAZY) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -144,13 +185,51 @@ This section describes required fields for the json configuration file.
...
+    sudo docker exec alphartc peerconnection_serverless sender.json
+    ```
+
+- Bare metal
+
+    If you compiled your own binary, you can also run it on your bare-metal machine.
+    
+    - Linux users:
+        1. Copy the provided corpus to a new directory
+            ```shell
```
Comment (Pterosaur) [inline README.md:203]: Add empty line before ```code```. You can install https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one on VSCode to check some Markdown warning.

**Comment by Pterosaur** [comment]
win_gui cannot work

File: modules/remote_bitrate_estimator/remote_estimator_proxy.cc [hunk 3]
```diff
@@ -93,7 +93,7 @@ void RemoteEstimatorProxy::IncomingPacket(int64_t arrival_time_ms,
 
   //--- BandWidthControl: Send back bandwidth estimation into to sender ---
   bool time_to_send_bew_message = TimeToSendBweMessage();
-  float estimation = 0;
+  float estimation = 8*1e6;
```
Comment (Pterosaur) [inline modules/remote_bitrate_estimator/remote_estimator_proxy.cc:96]: Please recover original version. It's my fault to submit a wrong value.

**Review by Pterosaur — APPROVED** [review]

**Pull request merged**